public/index.css

```diff
@@ -202,3 +202,57 @@ body {
     padding: 5% 25%;
   }
 }
+
+/* footer styles  */
+
+.mhi-footer {
+  /* display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between; */
+  font-size: 0.77777em;
+  padding-left: 25px;
+  padding-right: 25px;
+  /* background-color: #f9f9f9; */
+  padding-top: 2.5rem;
+}
+
+.footer-header {
+  position: relative;
+  width: 180px;
+  left: 50%;
+  transform: translate(-50%);
+}
+
+.footer-header > h4 > svg {
+  transform: translate(0, 25%);
+}
+.mhi-logo {
+  position: relative;
+  /* padding: 0.5rem 0; */
+  width: 135px;
+  left: 50%;
+  transform: translate(-50%);
+}
+
+.footer-banner {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  background-color: var(--human-blue);
+  width: 100%;
+  margin-top: 5rem;
+  height: 3.5rem;
+  padding: 0 1rem;
+}
+
+.footer-links {
+  min-width: 160px;
+}
```

Comment:
wanted to prevent the footer links wraps like this. If we not mind the wrap,  I will removed it. Thanks!
![Screen Shot 2021-01-01 at 12 11 10 PM](https://user-images.githubusercontent.com/16155615/103443179-89d97780-4c2a-11eb-822b-b2c2132a44f5.png)

Reply:
decided to keep the mini-width, where icons will stay straight on smaller screen.